FontMenuNameDB

```diff
@@ -5,17 +5,84 @@
 #   m=1,Macintosh_Compatible_MenuName
 
 
-# Variable fonts -----------------------------
+# Variable fonts --------------------------------
 
-[SourceSerifVariable-Roman]
-    f=Source Serif Variable
+[SourceSerif4Variable-Roman]
+    f=Source Serif 4 Variable
     s=Roman
 
```

Comment:
Would it be possible to add a `l=Source Serif 4 Variable` here – or is style linking across VFs not (yet) possible?

Reply:
Style linking can be handled a couple ways in the STAT, but it's not entirely clear how all apps are handling this.